pkg/cloudprovider/aws/fleet/packing/packing.go

```diff
@@ -45,12 +47,108 @@ func NewPacker(ec2 ec2iface.EC2API) *PodPacker {
 // Pack returns the packings for the provided pods. Computes a set of viable
 // instance types for each packing of pods. Instance variety enables EC2 to make
 // better cost and availability decisions.
+// Pods provided are all schedulable in the same zone as tightly as possible.
 func (p *PodPacker) Pack(ctx context.Context, pods []*v1.Pod) ([]*Packings, error) {
 	zap.S().Debugf("Successfully packed %d pods onto %d nodes", len(pods), 1)
-	return []*Packings{
-		{
-			InstanceTypeOptions: []string{"m5.large"}, // TODO, prioritize possible instance types
-			Pods:                pods,
-		},
-	}, nil
+
+	// 1. Arrange pods in decreasing order by the amount of CPU requested, if
```

Comment:
Might be worth linking to https://en.wikipedia.org/wiki/Bin_packing_problem#First_Fit_Decreasing_(FFD)